src/TLuaInterpreter.cpp

```diff
@@ -2456,7 +2458,7 @@ int TLuaInterpreter::getFont(lua_State* L)
             windowName = QString::fromUtf8(lua_tostring(L, 1));
 
             if (windowName.isEmpty() || windowName.compare(QStringLiteral("main"), Qt::CaseSensitive) == 0) {
-                font = pHost->mDisplayFont.family();
+                font = mudlet::self()->mConsoleMap[pHost]->mUpperPane->fontInfo().family();
```

Comment:
Why is it necessary to reach up into the `mudlet` class to get access to the main profile `TConsole` instance - what is wrong with using the `Host` pointer to the same thing?